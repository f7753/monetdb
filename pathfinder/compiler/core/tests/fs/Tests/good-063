#!/bin/bash

pf -s9 <<'EOT'
doc("foo.xml")/item
EOT
echo $?
