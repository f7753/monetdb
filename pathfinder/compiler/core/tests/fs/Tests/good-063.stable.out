stdout of test 'good-063` in directory 'compiler/core/tests/fs` itself:


# 15:46:31 >  
# 15:46:31 >  Mtimeout -timeout 60 ./good-063 good-063 
# 15:46:31 >  

0

# 15:46:31 >  
# 15:46:31 >  Done.
# 15:46:31 >  

